src/landscape.cpp

```diff
@@ -738,6 +738,8 @@ CommandCost CmdLandscapeClear(TileIndex tile, DoCommandFlag flags, uint32 p1, ui
  * @param p1 start tile of area dragging
  * @param p2 various bitstuffed data.
  *  bit      0: Whether to use the Orthogonal (0) or Diagonal (1) iterator.
+ *  bit      1: Whether to clear all tiles (0) or only the tiles with trees (1).
+ *  bit      2: Whethet ro clear all tiles (0) or only the tiles owned by the company (1).
```

Comment:
I'd suggest making this an enumeration (8 bit wide) for "mode" instead of a series of bits that can't be set in combination anyway.